chart/kepler/templates/server.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       serviceAccountName: {{ include "kepler.serviceAccountName" . }}
       containers:
       - name: kepler-exporter
-        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
is it `.Chart.appVersion` or `.Chart.AppVersion`

Reply:
That is a great question.

I always copy-paste from other chart, like: [here](https://github.com/prometheus-community/helm-charts/blob/4fa4681e87838861bcc38b77114c39f6aff61eea/charts/prometheus-smartctl-exporter/templates/daemonset.yaml#L39).

Also if you do a `helm create example`, it uses ` image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"`

Reply:
good to know